src/main/java/com/revature/services/ResultSummaryService.java

```diff
@@ -0,0 +1,47 @@
+package com.revature.services;
+
+import com.revature.models.ResultSummary;
+import com.revature.models.SwaggerSummary;
+import com.revature.repositories.ResultSummaryRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class ResultSummaryService {
+
+    @Autowired
+    private ResultSummaryRepository repository;
+
+    public ResultSummaryService(ResultSummaryRepository mockedDao) {
+        this.repository = mockedDao;
+    }
+
+    public ResultSummary insert(ResultSummary rs) {
+        return repository.save(rs);
+    }
+
+    public boolean update(ResultSummary s) {
```

Comment:
Rather minor, but for these methods, the final `else` block isn't needed. If line 28 is executed, then the following code won't be executed anyways. It would look a bit cleaner if the `else` was therefore removed, and `return false` was just a statement after the `if`.